packages/client/wallets/smart-wallet/src/blockchain/chains.ts

```diff
@@ -61,6 +61,6 @@ export const viemNetworks: Record<SmartWalletChain, Chain> = {
     "arbitrum-sepolia": arbitrumSepolia,
 };
 
-export const getBundlerRPC = (chain: SmartWalletChain) => {
+export function getBundlerRPC(chain: SmartWalletChain) {
```

Comment:
let's be consistent–if we use `function` at all, let's make it so all functions use it rather than `const + arrow`

Reply:
sgtm